sqs_workers/queue.py

```diff
@@ -581,7 +581,14 @@ def process_messages(self, messages: List[Any]) -> bool:
         return all(results)
 
     def process_message_fallback(self, job_name):
-        logger.warning("Error while processing %s.%s", self.name, job_name)
+        # Note: we don't set exc_info=True, since source of the error is almost
+        # certainly in another code base.
+        logger.error(
+            "Could not find an SQS processor for %s.%s."
+            "Has it been registered correctly?",
```

Comment:
nit: there should be a space at the end of the first string

Reply:
Ah good catch 🎣. Next time I'll wait for your review before merging🤦.

Reply:
No worries 😊 And no need to wait either if you already have a ✅, that's mostly me being a bit of a perfectionist 😅